easybuild/easyconfigs/e/EvidentialGene/EvidentialGene-2018.01.01-intel-2017a-Perl-5.24.1.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = "PackedBinary"
+
+name = "EvidentialGene"
+version = '2018.01.01'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://arthropods.eugenes.org/EvidentialGene/'
+description = """EvidentialGene is a genome informatics project for
+ "Evidence Directed Gene Construction for Eukaryotes",
+ for constructing high quality, accurate gene sets for
+ animals and plants (any eukaryotes), being developed by
+ Don Gilbert at Indiana University, gilbertd at indiana edu."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = [
+    'http://arthropods.eugenes.org/EvidentialGene/other/evigene_old/',
+    'http://arthropods.eugenes.org/EvidentialGene/other/evigene_old/evigene_older/',
+]
+month = ['', 'jan', 'feb', 'mar', 'apr', 'may', 'jun', 'jul', 'aug', 'sep', 'oct', 'nov', 'dec']
+dlver = '%(version_major)s'[-2:] + month[int('%(version_minor)s') - 1] + version.split('.')[2]
```

Comment:
@hajgato This doesn't work, it breaks parsing of the easyconfig file:

```
Parsing easyconfig file failed: invalid literal for int() with base 10: '%(version_minor)s' (line 21)"
```

The template values can only be replaced *after* parsing the easyconfig file, so the `int(...)` is getting a literal `'%(version_minor)s` string...

Reply:
@boegel Fixed